crates/extension/src/wasm_host/wit.rs

```diff
@@ -29,7 +29,7 @@ pub fn new_linker(
     f: impl Fn(&mut Linker<WasmState>, fn(&mut WasmState) -> &mut WasmState) -> Result<()>,
 ) -> Linker<WasmState> {
     let mut linker = Linker::new(&wasm_engine());
-    wasmtime_wasi::command::add_to_linker(&mut linker).unwrap();
+    wasmtime_wasi::add_to_linker_async(&mut linker).unwrap();
```

Comment:
Updated the `add_to_linker` usage based on this example:

https://github.com/bytecodealliance/wasmtime/pull/8228/files#diff-7300ada624384b2aedf83078d6e952a1a05f9c9bb6257a2d4a0266e8c35ae7deR16